Storage/Storage/OpenTabs.swift

```diff
@@ -0,0 +1,58 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+* License, v. 2.0. If a copy of the MPL was not distributed with this
+* file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+//
+
+import Foundation
+
+public protocol OpenTabs {
```

Comment:
Naming OCD: I'd rather go with SessionStorage (like Desktop and Android) or similar. This storage will also contain closed tabs so we can undo closing a tab. Form data and other bits might make their way into the storage too.